### DISCUSSION TITLE
Interpreting Links in a coverage plot

### DISCUSSION THREAD

**Discussion by aleenah23** [stuart-lab/signac#1418] [Q&A] [answered]
Hi, 

I have recently started using Signac and I am facing some difficulty in interpreting the coverage plots. I have multimodal data from multiple samples that I have jointly analysed. 

I ran the LinkPeaks function on the combined object then visualised the coverage plot with both the gene expression and peaks. In the "Links" section, I am a little bit confused regarding what it's showing. Based on the documentations and a discussion with a colleague, we assume the following:

- At every peak for a specific gene, the relationship between gene expression and accessibility is calculated. We end up with a correlation coefficient/ expected coefficient.
- Peaks with similar coefficients are linked.
- Linked peaks ultimately are the ones that strongly correlate with gene expression and therefore most likely regulate it.
- The intensity of the blue linking lines is indicative of the strength of correlation. Based on the documentation we assume that is possibly based on the z-score.

I was just wondering if this is a correct way to interpret/ read the plot.  Any feedback or insights would be appreciated.

Thanks,
Aleen

**Comment by timoast** [comment, marked as answer]
> At every peak for a specific gene, the relationship between gene expression and accessibility is calculated. We end up with a correlation coefficient/ expected coefficient.

Correct

> Peaks with similar coefficients are linked.

No, not sure what this means. Peaks-gene pairs with a correlation are linked.

> Linked peaks ultimately are the ones that strongly correlate with gene expression and therefore most likely regulate it.

Yes that's the goal

> The intensity of the blue linking lines is indicative of the strength of correlation. Based on the documentation we assume that is possibly based on the z-score.

Yes (score is the correlation coefficient)

Please check the Signac paper for more information: https://www.nature.com/articles/s41592-021-01282-5

**Reply by aleenah23** [reply]
Thanks very much!
